aidants_connect_web/models.py

```diff
@@ -341,10 +348,44 @@ def remove_from_organisation(self, organisation: Organisation) -> Optional[bool]
             return self.is_active
 
         self.organisations.remove(organisation)
-        if self.organisation not in self.organisations.all():
+        if not self.is_in_organisation(organisation):
```

Comment:
Tu ne voulais pas plutôt faire ceci ?

```suggestion
        if not self.is_in_organisation(self.organisation):
```

Puisqu'on vient de faire `self.organisations.remove(organisation)`, alors `self.is_in_organisation(organisation)` est toujours faux.

Reply:
Mais c'est très vrai, ça, dis :thinking:

Reply:
Bonne attape… Les tests on complètement contourné ça :astonished: